install.dd

```diff
@@ -0,0 +1,200 @@
+    )
+ )
+ $(LI For `ldc`, installation of the following versions is supported:
+    $(UL
+        $(LI `ldc` - latest stable version)
+        $(LI `ldc-beta` - latest beta version)
+        $(LI `ldc-<version>` - specific official release (including beta releases))
+    )
+ )
+ $(LI For `gdc`, installation of the following versions is supported:
+    $(UL
+        $(LI `gdc` - latest stable version)
+    )
+ )
+)
+
+$(H3 Content)
+
+$(UL
+    $(LI $(RELATIVE_LINK2 get, Downloading the installer))
+    $(LI $(RELATIVE_LINK2 usage, Usage))
+    $(UL
+        $(LI $(RELATIVE_LINK2 install, Installation))
+        $(LI $(RELATIVE_LINK2 uninstall, Uninstallation))
+        $(LI $(RELATIVE_LINK2 list, Listing available compilers))
+        $(LI $(RELATIVE_LINK2 update, Update))
+    )
+)
+
+$(H2 $(LNAME2 get, Downloading the installer))
+
+$(CONSOLE
+wget https://dlang.org/install.sh
+)
+
+Alternatively, the script can be invoked directly:
+
+$(CONSOLE
+curl https://dlang.org/install.sh | bash -s
+)
+
+$(P If no argument are provided, the latest DMD compiler will be installed.)
+
+$(P The installer will also install a copy of itself to `~/dlang/install.sh`.)
+
+$(H2 $(LNAME2 usage, Usage))
+
+$(CONSOLE
+~/dlang/install.sh [&lt;command&gt;] [&lt;args&gt;]
+)
+
+$(UL
+    $(LI $(RELATIVE_LINK2 install, Installation))
+    $(LI $(RELATIVE_LINK2 uninstall, Uninstallation))
+    $(LI $(RELATIVE_LINK2 list, Listing available compilers))
+    $(LI $(RELATIVE_LINK2 update, Update))
+)
+
+$(H3 $(LNAME2 global_options, Global options))
+
+$(DL
+    $(DT
+        $(SWITCH $(SWNAME -p) $(SWNAME --path),
+            Changes the install location (default `~/dlang`))
+        $(SWITCH $(SWNAME -v),
+            Activates more verbose output)
+        $(SWITCH $(SWNAME -h),
+            Displays a help page)
+    )
+)
+
+$(H3 $(LNAME2 install, Install))
+
+$(CONSOLE
+~/dlang/install.sh install &lt;compiler&gt;
+)
+
+Download and install a D compiler.
+By default the latest release of the DMD compiler is selected.
+
+$(H4 Options)
+
+$(DL
+    $(DT
+        $(SWITCH $(SWNAME -a) $(SWNAME --active),
+            Only prints the path to the activate script)
+        $(SWITCH $(SWNAME dmd|ldc|gdc),
+            Installs the latest version of a compiler)
+        $(SWITCH $(SWNAME dmd|ldc|gdc-&lt;version&gt;),
+            Installs a specific version of a compiler (e.g. dmd-2.071.1, ldc-1.1.0-beta2))
+        $(SWITCH $(SWNAME dmd|ldc-beta),
+            Installs the latest beta version of a compiler)
+        $(SWITCH $(SWNAME dmd-nightly),
+            Installs DMD nightly)
+        $(SWITCH $(SWNAME dmd-20170-02-10),
+            Installs specific DMD nightly)
+    )
+)
+
+$(EXAMPLES
+~/dlang/install.sh
+~/dlang/install.sh dmd
+~/dlang/install.sh install dmd
+~/dlang/install.sh install dmd-2.071.1
+~/dlang/install.sh install ldc-1.1.0-beta2
+)
+
+$(P Once a compiler can be activated for the current session:)
```

Comment:
Why put this in its own paragraph?  Merge it with the next sentence instead `session, run the`